testworld/incorrect_proto_test.go

```diff
@@ -0,0 +1,113 @@
+// +build testworld
+
+package testworld
+
+import (
+	"github.com/centrifuge/go-centrifuge/config"
+	"github.com/centrifuge/go-centrifuge/testingutils/config"
+
+	"testing"
```

Comment:
could you organise imports according to https://github.com/golang/go/wiki/CodeReviewComments#imports ? Its the convention we follow. Group standard lib imports first and then the others (like in the other files above ).
Unfortunately `goimports` does not do it sometimes.

Reply:
oh right, of course.